src/raftstore/store/peer.rs

```diff
@@ -401,7 +425,8 @@ impl Peer {
         if req.has_admin_request() {
             match req.get_admin_request().get_cmd_type() {
                 AdminCmdType::CompactLog |
-                AdminCmdType::InvalidAdmin => {}
+                AdminCmdType::InvalidAdmin |
+                AdminCmdType::TransferLeader => {}
```

Comment:
"same" means I haven't thought some cases may trouble us much, when split happens.
ok, I can add epoch check just in case.